src/assets/locales/resources_en_US.json

```diff
@@ -19,6 +19,10 @@
     "_description": "Text for modal close button",
     "message": "Close modal"
   },
+  "skyux_modal_close_confirm_message": {
+    "_description": "Text for modal exit confirm dialog's message",
+    "message": "Are you sure you want to close this dialog?"
+  },
```

Comment:
@Blackbaud-ToddRoberts When you get the chance could you weigh in on this default message for confirming the close of a modal? Just want to make sure that this is a good default. Matt said it looked alright in our slack messaging, but I want to make sure I hear from both of you

Reply:
That seems fine to me

Reply:
Sweet, thank you :)